Gopkg.toml

```diff
@@ -12,17 +12,45 @@ required = ["github.com/docker/distribution"]
   name = "github.com/docker/distribution"
   revision = "b38e5838b7b2f2ad48e06ec4b500011976080621"
 
+#
+# The most significant dependency for the Kubernetes monitor: the controller-runtime
+# NOTE: change with care and always adjust the Kubernetes dependencies below
+#
+[[override]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "v0.1.10"
+
+#
+# Kubernetes dependencies
+# NOTE: always match exactly to what controller-runtime uses
+#
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.13.1"
```

Comment:
Just wondering if we have any backward compatibility issue with 1.11 if we stick to 1.13 clients.

Reply:
The operator is also using this specific version of kubernetes. For the controller-runtime it is very important to match the Kubernetes version to what the controller-runtime is using internally. I always match them against that for the least problems.

In the operator I tested everything down to 1.9, and have not seen any issues yet. We're not using any extravagant features in the controller in the monitor, so I don't think that there is going to be issues.